cronjob.yml

```diff
@@ -0,0 +1,23 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+ name: hello-cronjob
+ namespace: mateapp
+spec:
+ schedule: "*/4 * * * *"
+ concurrencyPolicy: Allow
+ startingDeadlineSeconds: 100
+ successfulJobsHistoryLimit: 10
+ failedJobsHistoryLimit: 5
+ jobTemplate:
+   spec:
+     template:
+       spec:
+         containers:
+           - name: todoapp
+             image: ikulyk404/busyboxplus:curl
+             command:
+               - /bin/sh
+               - -c
+               - curl http://172.18.0.4:30008/api/health
```

Comment:
Ensure that the URL 'http://172.18.0.4:30008/api/health' is correct and accessible from the pod. If the IP or port changes, the command might fail.